kernel/util.c

```diff
@@ -73,7 +87,11 @@ bool try_set_access_flag(unsigned long addr)
 out_pte_unlock:
 	pte_unmap_unlock(ptep, ptl);
 out_unlock:
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 4, 0)
```

Comment:
```suggestion
#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0)
```

first introduced at 5.8
https://elixir.bootlin.com/linux/v5.8/A/ident/mmap_read_unlock

Reply:
https://github.com/aosp-mirror/kernel_common/commit/b524137fa1d8cd9aabb65fbf34da10e24f8fc9d7
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/Makefile?h=v5.4.208&id=b524137fa1d8cd9aabb65fbf34da10e24f8fc9d7
Linux kernel backport this feature in 5.4.208

Reply:
> > first introduced at 5.8 https://elixir.bootlin.com/linux/v5.8/A/ident/mmap_read_unlock
> 
> > [aosp-mirror/kernel_common@b524137](https://github.com/aosp-mirror/kernel_common/commit/b524137fa1d8cd9aabb65fbf34da10e24f8fc9d7) https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/Makefile?h=v5.4.208&id=b524137fa1d8cd9aabb65fbf34da10e24f8fc9d7 Linux kernel backport this feature in 5.4.208
> 
> This feature was not backported to kernel versions 5.5 and 5.6. For versions between 5.4.208 and 5.5, should I implement special handling, or simply treat them as if the feature is unavailable?

no real android devices use these version
android devices only use LTS